webapp/src/websocket_handlers.ts

```diff
@@ -182,6 +196,8 @@ export async function handleUserJoined(store: Store, ev: WebSocketMessage<UserJo
         notificationsStopRinging(); // And stop ringing for _any_ incoming call.
     }
 
+    store.dispatch(loadProfilesByIdsIfMissing([userID]));
```

Comment:
Given the comments we're adding about everything being synchronous, maybe we should add a comment that this will dispatch will be async?